rotkehlchen/chain/evm/names.py

```diff
@@ -24,6 +25,13 @@
     from rotkehlchen.chain.ethereum.node_inquirer import EthereumInquirer
 
 
+GLOBAL_ADDRESSBOOK_PRIORITIES = {
+    GlobalAddressbookSource.MANUAL: 0,
+    GlobalAddressbookSource.CURVE_CACHE: 1,
+    GlobalAddressbookSource.WALLET_LABELS_XYZ: 2,
```

Comment:
Can you also specify in a comment if 0 is highest priority or lowest? I know is sorting from lower to higher later but is good to have it here to avoid misunderstandings later